bin/supply

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+echo "-----> DatadogBuildpack/supply"
+
+BIN_DIR=$(cd $(dirname $0); pwd)
+ROOT_DIR=$(dirname $BIN_DIR)
+BUILD_DIR=$1
+CACHE_DIR=$2
+ENV_DIR=$3
+
+echo "Installing Datadog Dogstatsd and Trace Agent"
+
+mkdir -p $BUILD_DIR/datadog/scripts
+mkdir -p $BUILD_DIR/.profile.d
+
+cp -r $ROOT_DIR/lib/dist $BUILD_DIR/datadog/
+cp -r $ROOT_DIR/lib/conf.d $BUILD_DIR/datadog/
+if [ -f $ROOT_DIR/lib/puppy ]; then
+  cp $ROOT_DIR/lib/puppy $BUILD_DIR/datadog/puppy
+fi
+if [ -f $ROOT_DIR/lib/dogstatsd ]; then
+  cp $ROOT_DIR/lib/dogstatsd $BUILD_DIR/datadog/dogstatsd
+fi
+cp $ROOT_DIR/lib/dogstatsd $BUILD_DIR/datadog/dogstatsd
```

Comment:
we should remove this since we have the check above to see if it exists (this is my error but we shouldn't carry it forward)